content_scripts/link_hints.coffee

```diff
@@ -372,7 +372,7 @@ class LinkHintsMode
             Utils.nextTick -> focusThisFrame highlight: true
           else if localHintDescriptor.reason == "Scroll."
             # Tell the scroller that this is the activated element.
-            handlerStack.bubbleEvent "DOMActivate", target: clickEl
+            handlerStack.bubbleEvent (if Utils.isFirefox() then "click" else "DOMActivate"), target: clickEl
```

Comment:
Where did this change come from? Seems unrelated to this PR.

Reply:
Unrelated. This is just to fix another issue that on Firefox `LinkHints` can not notify `Scroller` what to scroll.